ngraph/core/src/op/space_to_depth.cpp

```diff
@@ -226,7 +226,7 @@ bool ngraph::op::v0::SpaceToDepth::evaluate(const HostTensorVector& outputs,
 namespace ngraph
 {
     template <>
-    EnumNames<op::v0::SpaceToDepth::SpaceToDepthMode>&
+    NGRAPH_API EnumNames<op::v0::SpaceToDepth::SpaceToDepthMode>&
```

Comment:
This enum is used only inside this cpp file and I hope no one includes cpp files :)
So which problem this NGRAPH_API solves?

Reply:
we used ```ngraph::as_string``` when print error message, if remove ```NGRAPH_API``` from .cpp and build in Release mode we got linker error "undefined reference", in Debug mode this problem is not reproduced
https://github.com/dmitry-gorokhov/openvino/blob/feature/cpu_migration_on_ngraph/inference-engine/src/mkldnn_plugin/nodes/mkldnn_space_to_depth_node.cpp#L33

Reply:
Needs to be discussed offline. Lets fix in separate PR (if necessary).